activiti-spring-boot/activiti-spring-boot-test/.gitignore

```diff
@@ -0,0 +1,13 @@
+/target
```

Comment:
I think that's a bad idea adding `.gitignore` inside every subproject (unless that there is some specific configuration for this subproject, but It does not seem to be the case here).